static/js/util/util.js

```diff
@@ -228,40 +228,33 @@ complete profile consists of:
 */
 export function validateProfileComplete(profile) {
   let reqFields = [];
+  let validationMessages = {};
 
   // check personal tab
   reqFields = reqFields.concat(...PersonalTab.defaultProps.requiredFields);
-  let errors = validateProfile(profile, reqFields, {});
+  _.assignIn(validationMessages, PersonalTab.defaultProps.validationMessages);
+
+  let errors = validateProfile(profile, reqFields, validationMessages);
   if ( !_.isEqual(errors, {}) ) {
-    return ([false, {
-      url: "/profile/personal",
-      title: "Personal Info",
-      text: "Please complete your personal information.",
-    }]);
+    return ([false, "/profile/personal", errors]);
   }
```

Comment:
can you add a check for completeness of the education tab? you should be able to follow the same pattern we've got here for the others.